src/utils.js

```diff
@@ -658,23 +653,21 @@ function suggestCorrectedDateFormat(inputDate) {
 /**
  * Parses a duration string representing a past time window and converts it to milliseconds.
  *
- * @param {string} pastTimeWindow - The past time duration to parse, e.g., "20 mins", "15 minutes".
+ * @param {string} pastTimeWindow - The past time duration to parse, e.g., "20", "15".
  * @returns {number} The duration in milliseconds.
  */
 function parsePastDuration(pastTimeWindow) {
-	// Regular expression to match various formats of minute abbreviations
-	const pastDurationRegex = /^(\d+)\s*(m|mins?|minutes?)$/i;
-	const match = pastTimeWindow.match(pastDurationRegex);
+	// Check if pastTimeWindow contains non-numeric characters
+	const match = pastTimeWindow.match(/^(\d+)$/);
 
-	if (!match) {
+	const durationInMs = parseInt(pastTimeWindow) * 60 * 1000;
```

Comment:
Recommend using Number() here if the expectation is that the entire string is a number.
```suggestion
	const durationInMs = Number(pastTimeWindow) * 60 * 1000;
```